.buildkite/integration.pipeline.yml

```diff
@@ -2,6 +2,12 @@
 
 env:
   VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+  ASDF_MAGE_VERSION: 1.14.0
+
+  # The following images are defined here and their values will be updated by updatecli
+  # Please do not change them manually.
+  IMAGE_UBUNTU_2004_X86_64: "platform-ingest-elastic-agent-ubuntu-2004-1752886846"
+  IMAGE_UBUNTU_2004_ARM_64: "platform-ingest-elastic-agent-ubuntu-2004-aarch64-1752886846"
```

Comment:
We can use `ubuntu-2204` too, I've just found that `ubuntu-24` uses the latest version instead of the fixed version as in 22 or 20. 

>  Docker <25 is only available before Ubuntu 24.04. Install latest on Ubuntu 24.04


Let's bump the version of ubuntu-22, since `ubuntu-20` reached EoL in May 2025

<img width="1009" height="254" alt="image" src="https://github.com/user-attachments/assets/6f633035-1a32-4242-95be-e864858735b1" />


```suggestion
  IMAGE_UBUNTU_2204_X86_64: "platform-ingest-elastic-agent-ubuntu-2204-1752886846"
  IMAGE_UBUNTU_2204_ARM_64: "platform-ingest-elastic-agent-ubuntu-2204-aarch64-1752886846"
```